docs/2.3/_modules/torch.html

```diff
@@ -2281,7 +2281,7 @@ <h1>Source code for torch</h1><div class="highlight"><pre>
 
 <span class="sd">        - Experimental or debug in-tree backends can be seen with `torch._dynamo.list_backends(None)`</span>
 
-<span class="sd">        - To register an out-of-tree custom backend: https://pytorch.org/docs/main/compile/custom-backends.html</span>
+<span class="sd">        - To register an out-of-tree custom backend: https://pytorch.org/docs/stable/torch.compiler_custom_backends.html</span>
```

Comment:
main/compile/custom-backends.html changed to stable/torch.compiler_custom_backends.html